lib/src/core/signal_client.dart

```diff
@@ -240,6 +256,47 @@ class SignalClient extends Disposable with EventsEmittable<SignalEvent> {
       _updateConnectionState(ConnectionState.disconnected);
     }
   }
+
+  void _sendPing() {
+    _sendRequest(lk_rtc.SignalRequest()
```

Comment:
I think we need to call `_resetPingTimeout` here; each time we send a ping. otherwise subsequent ping timeouts would not be detected.

Reply:
that makes sense